ironfish-cli/src/utils/spendPostTime.ts

```diff
@@ -50,6 +32,11 @@ async function benchmarkSpendPostTime(client: RpcClient, account: string): Promi
 
   const notes = await fetchNotes(client, account, 10)
 
+  // Not enough notes in the account to measure the time to combine a note
+  if (notes.length < 3) {
+    return 0
```

Comment:
I didn't look through the code to see where this is used, but do we handle this new 0 case somewhere up the stack so it doesn't look like your post time is instant?

Also, unrelated to this PR, but you could just make fake notes to do the benchmarking rather than requiring the client to have a certain number of notes.

Reply:
This check happens upstream in `fetchNotes` in the `wallet:notes:combine` command

Reply:
I added an exception to be extra sure that this case doesn't get hit.